Deploy.yml

```diff
@@ -0,0 +1,35 @@
+---
+- name: Deployee Employee-portal
+  hosts: all
+  become: yes
+  become_user: root
+  become_method: sudo
+  remote_user: ubuntu
+  tasks:
+    - name: Run apt-update and apt-upgrade
+      apt:
+          upgrade: yes
+          update_cache: yes
+    - name: Install docker-io
+      apt:
+          name: docker.io
+          state: latest
+          update_cache: true
+    - name: Install docker-compose from official github repo
+      get_url:
+        url: https://github.com/docker/compose/releases/download/1.29.2/docker-compose-Linux-x86_64
+        dest: /usr/local/bin/docker-compose
+        mode: 'u+x,g+x'
+    - name: Copy file with owner and permissions
+      ansible.builtin.copy:
+        src: ./docker-compose/
+        dest: ./
+    - name: Show directory
+      shell:  ls
+      register: output
+    - debug:
+        var: output.stdout_lines  
+    - name: Docker compose down
```

Comment:
This might cause significant downtime. Is there a way to just stop the server and pull the new image and restart it?